docs/src/content/docs/options.md

```diff
@@ -268,3 +268,9 @@ When creating HTML archive snapshots, pass additional options to the `single-fil
 See `single-file --help` for complete list of arguments, or browse source: https://github.com/gildas-lormeau/single-file-cli/blob/master/options.js
 
 Example: `LD_SINGLEFILE_OPTIONS=--user-agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:124.0) Gecko/20100101 Firefox/124.0"`
+
+### `LD_DISABLE_REQUEST_LOGS`
+
+Values: `true` or `false` | Default =  `false`
```

Comment:
Technically this seems to be active as long as the env variable is defined in some way, regardless of the value. So setting it to `false` will actually activate it. 

So maybe this should not list possible values, but rather state that whenever this env variable is defined with any value, then request logs will be disabled. Or investigate if the syntax in uwsgi.ini allows checking for specific values.

Reply:
This seems to do it:
```
if-env = LD_DISABLE_LOGGING=true
disable-logging = true
log-4xx = true
log-5xx = true
endif =
```